releases.py

```diff
@@ -191,13 +195,19 @@ def custom_sort_release(self, a, b):
         if not a_maj_min_patch:
             a_maj_min_patch = self._regex_xydate_custom_short_sort.search(a)
 
+        if not a_maj_min_patch:
+            raise Exception('ERROR: Filename not like expected')
```

Comment:
An f-string, or similar, to make this exception more useful:

`raise Exception(f'ERROR: Unexpected filename: {a}')`